examples/throttle.js

```diff
@@ -20,10 +20,21 @@ export const options = {
       },
       exec: 'throttled',
     },
+    cpuThrottled: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+      exec: 'cpuThrottled',
+    },
   },
   thresholds: {
     'browser_http_req_duration{scenario:normal}': ['p(99)<500'],
     'browser_http_req_duration{scenario:throttled}': ['p(99)<1500'],
+    'iteration_duration{scenario:normal}': ['p(99)<3000'],
+    'iteration_duration{scenario:cpuThrottled}': ['p(99)<5000'],
```

Comment:
Is this good enough for now? It's difficult to find the right sweet spot where this threshold will hold true on all platforms. It seems to fail on the Windows CI runs, so i need to fine tune it still.